noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -202,9 +203,10 @@ impl PrivateContext {
     }
 
     // docs:start:context_message_portal
-    pub fn message_portal(&mut self, content: Field) {
-        // docs:end:context_message_portal
-        self.new_l2_to_l1_msgs.push(content);
+    pub fn message_portal(&mut self, recipient: EthAddress, content: Field) {
+        // docs:end:context_message_portal {
```

Comment:
The `{` don't need to be here

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4512/commits/f54ef2d7d7bc31881c1f76b95214d91e71adca9b